chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/ComplaintDAOImpl.java

```diff
@@ -35,14 +35,9 @@
 @Component
 public class ComplaintDAOImpl extends BaseDAOImpl implements ComplaintDAO {
     private static final Logger LOGGER = LogManager.getLogger(ComplaintDAOImpl.class);
-    private static final String GET_COMPLAINTS_HQL = "SELECT DISTINCT c "
-            + "FROM ComplaintEntity c "
-            + "LEFT JOIN FETCH c.listings "
-            + "LEFT JOIN FETCH c.surveillances "
-            + "LEFT JOIN FETCH c.criteria "
-            + "JOIN FETCH c.certificationBody "
-            + "JOIN FETCH c.complainantType "
-            + "JOIN FETCH c.complaintStatusType "
+    private static final String GET_COMPLAINTS_HQL = "SELECT DISTINCT c " + "FROM ComplaintEntity c "
```

Comment:
Is this just because of Eclipse auto-formatting? I feel like it was easier to read the other way...